doxytag2zealdb/doxytag.py

```diff
@@ -134,10 +134,25 @@ def get_filename(self, tag):
             A string that would be appropriate to use as the documentation
             filename for an entry in a Zeal database.
         '''
+        def get_first_content_or_none(el):
```

Comment:
Since this method is just used locally and the code is clear, here are some things to consider:

1) Convert it to a one-line lambda
2) Keep as a function but remove docstring

Feel free to keep it as-is, though.